contributing/getting-started-e2e.md

```diff
@@ -184,13 +184,20 @@ Running the `gulp` command will compile the code and start up a Node.js server l
 
 You can browse the [http://localhost:8000/examples](http://localhost:8000/examples) directory to see some demo pages for various AMP components and combination of components.
 
-Note that by default each of the pages in the /examples directory actually uses the production version of AMP JavaScript; you can verify this by loading an example page in your browser, viewing the source and seeing that it is loading the AMP JavaScript from cdn.ampproject.org.
+Note that by default each of the pages in the /examples directory actually uses the unminified AMP JavaScript from your local server; you can verify this by loading an example page in your browser, viewing the source and seeing that it is loading the unminified AMP JavaScript from `http://localhost:8000`.
 
-For local development you will usually want to load the JS from your local server to test your changes.  You can do this by changing the URL suffix from .html to .max.html, e.g.
+You can change from where the AMP JavaScript is served and whether it is minified by hitting the `/serve_mode=<mode>` endpoint:
+- [http://localhost:8000/serve_mode=default](http://localhost:8000/serve_mode=default)
 
-* [http://localhost:8000/examples/article.amp.html](http://localhost:8000/examples/article.amp.html) loads an example page that uses AMP JS from cdn.ampproject.org
+  This is the default. Unminified AMP JavaScript is served from the local server.
```

Comment:
Let's add that : for local development you will usually want to serve unminified JS to test your changes.